backends/dpdk/README.md

```diff
@@ -35,6 +35,8 @@ To load the 'spec' file in dpdk follow the instructions in the
 - Parser Value Sets
 
 ### Unsupported PSA externs and features
+- egress parser, control, and deparser are not implemented, only
+  ingress parser, control, and deparser.  There is no traffic manager.
```

Comment:
If I understand correctly, "traffic manager" is described in the PSA specification as "Packet buffer and Replication Engine (PRE)". Is that correct?

Reply:
I have added another commit to this PR that mentions packet buffer and packet replication engine, to make the terms easier to find in the PSA spec.